src/dbcp/data_mart/br_election_data.py

```diff
@@ -91,7 +91,7 @@ def _create_county_commission_elections_long(
     comissioner_elections = comissioner_elections.rename(columns=rename_dict)
 
     assert ~comissioner_elections.duplicated(
-        subset=["county_id_fips", "election_id"]
+        subset=["county_id_fips", "election_id", "is_primary"]
```

Comment:
There are 8 county/election items that had duplicates that differed only in `is_primary`. To be honest I'm not sure if that is a data error or there are actually simultaneous primaries/elections possibly due to staggered elections for multi-seat races